client/jetpack-cloud/sections/partner-portal/assign-license-form/index.tsx

```diff
@@ -79,7 +79,12 @@ export default function AssignLicenseForm( { sites }: any ): ReactElement {
 					>
 						{ translate( 'Assign later' ) }
```

Comment:
The "assign later" button is a little bit pushed to the side on mobile

![Screenshot 2022-02-23 at 16 43 38](https://user-images.githubusercontent.com/3846700/155354085-84f70036-3179-4015-a706-f34ca34c3626.png)